docs/examples/04_charging_stations_example.py

```diff
@@ -0,0 +1,84 @@
+# %%
```

Comment:
for whatever reason, when i run `_convert_examples_to_notebooks.py` for this file, it just renders as one big cell, so i only saw the final plot at first. perhaps a simple formatting fix